kubernetes.go

```diff
@@ -0,0 +1,68 @@
+package kubernetes
+
+import (
+	"context"
+	"errors"
+	"path/filepath"
+
+	"github.com/loadimpact/k6/js/modules"
+	k8sTypes "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+const version = "v0.0.1"
+
+type Kubernetes struct {
+	Version     string
+	Client      *kubernetes.Clientset
+	meraOptions metav1.ListOptions
+	ctx         context.Context
+}
+
+type KubernetesOptions struct {
+	ConfigPath string
+}
+
+func (obj *Kubernetes) Init(options KubernetesOptions) error {
```

Comment:
Would this be better suited as a constructor function? If you rename the function XKubernetes, users could just `new Kubernetes(options)`

Reply:
@simskij  could you please give a more detailed example. I'm constantly getting JS error:
```
ERRO[0000] TypeError: Value is not a constructor
```

Reply:
Sure! https://github.com/k6io/xk6-distributed-tracing/blob/master/tracing.go 👍🏼